docs/guides/represent-quantum-computers.ipynb

```diff
@@ -473,7 +470,7 @@
    "id": "f8b516be-805d-4d0b-92f9-44d3fbe3b51f",
    "metadata": {},
    "source": [
-    "Because we included error information, the `VF2PostLayout` pass tries to find the optimal qubits to use, resulting in the same circuit that we found originally with the same physical qubits."
+    "Because error information was included, the `VF2PostLayout` pass tries to find the optimal qubits to use, resulting in the same circuit that was originally found with the same physical qubits."
```

Comment:
```suggestion
    "Because the target includes error information, the `VF2PostLayout` pass tries to find the optimal qubits to use, resulting in the same circuit that was originally found with the same physical qubits."
```

So we can keep the active voice